pandas/io/formats/style.py

```diff
@@ -204,6 +206,101 @@ def _repr_html_(self) -> str:
         """
         return self.render()
 
+    def set_tooltips(self, ttips: DataFrame) -> "Styler":
```

Comment:
can you add an entry for this in style.rst

Reply:
done, and `set_tooltips_class` also